io.github.plrigaux.sysd-manager.yaml

```diff
@@ -0,0 +1,64 @@
+id: io.github.plrigaux.sysd-manager
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+#runtime: org.freedesktop.Platform
+#runtime-version: '23.08'
+#sdk: org.freedesktop.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+- org.freedesktop.Sdk.Extension.llvm16
+command: sysd-manager
+finish-args:
+- --share=ipc
+- --socket=fallback-x11
+- --socket=wayland
+- --device=dri
+#- --filesystem=home
+- --filesystem=host
```

Comment:
why is host needed ?

Reply:
it doesn't have access to the host `/usr`. Did you test it?

Reply:
You need to look at `/run/host` when inside the Flatpak.

Reply:
Do you need `home` too ? If not use `host-os` which has less access than plain `host`. see https://docs.flatpak.org/en/latest/sandbox-permissions.html#filesystem-access

Reply:
> You need to look at `/run/host` when inside the Flatpak.

I know I tested it. When inside a flatpack the program point to /run/host/usr/lib/systemd/...